participant/models.py

```diff
@@ -74,6 +76,31 @@ def actions(self):
 
         return actions
 
+    def phishing_email_behaviour_and_actions(self, email_id):
+        """
+        Aggregates actions related to a phishing email and determines behaviour.
+
+        :param email_id: UUID - id of an email
+        :return: (STRING, LIST) - returns a tuple of participant behaviour and actions
+        """
+
+        def participant_behaviour(actions):
+            pb = ParticipantBehaviour.NEUTRAL
+            if [a for a in actions if a.get('action_type') in POSITIVE_ACTIONS]:
```

Comment:
if this is a list comprehension won't this only work on action[0]

Reply:
list comprehensions return a list regardless so this should work fine

Reply:
can actions contain both positive and negative, and if so, overwriting `pb` with negative is always what we want? If so then 👍 just making sure!

Reply:
spot on - negatives overrule positives